### DISCUSSION TITLE
use $wire.set in defer mode

### DISCUSSION THREAD

**Discussion by camiant** [livewire/livewire#2022] [4. General]
hi all,
there's a way to make `$wire.set(property, value)` work in `defer` mode?

no info found at [https://laravel-livewire.com/docs/2.x/alpine-js](https://laravel-livewire.com/docs/2.x/alpine-js) inside section `Interacting With Livewire From Alpine: $wire`

thanks for your support 👋🏻

**Comment by psos** [comment]
$wire.set(property, value, true)

https://github.com/livewire/livewire/blob/85480ea501c90bc7bf859ead19f0466d44eddef0/js/component/index.js#L138

**Reply by camiant** [reply]
thanks a lot, @psos !
